src/MassTransit/Logging/Diagnostics/DiagnosticActivityExtensions.cs

```diff
@@ -26,7 +26,7 @@ public static void AddConsumeContextTags(this Activity activity, ConsumeContext
         {
             if (activity.IsAllDataRequested)
             {
-                SetTag(activity, DiagnosticHeaders.MessageId, context.MessageId);
+                SetTag(activity, DiagnosticHeaders.Messaging.MessageId, context.MessageId);
```

Comment:
I believe messaging.message.id is meant for the _transport_ messageId, not MT's.

Reply:
Hmm, the description for `messaging.message.id` in the spec is: _A value used by the messaging system as an identifier for the message, represented as a string._

Reply:
Does MT generate an id? Does it depend on the transport (e.g. only if the transport doesn't generate it)? If MT generates an id, can it be different from the one in the transport?

Reply:
if context.MessageId is the MT id, do we have access to the transport id?

Reply:
Yes, MassTransit already puts the TransportMessageId in the messaging.message.id property.

Reply:
Ok, I see what you're referring to:

https://github.com/MassTransit/MassTransit/blob/282728da4ab31119e97f97865151a4d68b2b6317/src/MassTransit/Logging/Diagnostics/LogContextActivityExtensions.cs#L81-L84

I'll rollback the change for the message.id.

Reply:
> Hmm, the description for messaging.message.id in the spec is: A value used by the messaging system as an identifier for the message, represented as a string.

The id in `messaging.message.id` should make sense in the context of the system that is specified in `messaging.system`.